_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -0,0 +1,438 @@
+- are in the same [web page (HTML)][]; and
+- are [included in an accessibility tree][included in the accessibility tree]; and
+- have [matching][] [accessible names][accessible name] that are not empty (`""`); and
+- have the same [programmatically determined link context][].
+
+**Note:** The test target for this rule is the full set of link elements that share the same [matching][] [accessible name][] and [programmatically determined link context][].
+
+## Expectation
+
+When followed, the links in each set of target elements resolve to the [same resource][] or to [equivalent resources](#equivalent-resource). Resolving the links includes potential redirects, if the redirects happen instantly.
+
+## Assumptions
+
+- This rule assumes that the purpose of the links with identical [accessible names][accessible name] and [context][programmatically determined link context] would not be ambiguous to users in general, which is the exception mentioned in [Success Criterion 2.4.4 Link Purpose (In Context)][sc244]. If the links are ambiguous to users in general, users of assistive technologies are not at a disadvantage when viewing the links, which makes it more of a general user experience concern than an accessibility issue.
+- This rule assumes that, within the context of the test subject, the description provided by the [accessible name][] of a link can only accurately describe one resource (notably, homonyms alone are not used as link names). Thus, if two or more links have the same [accessible name][] but resolve to different resources, at least one of them does not describe its purpose.
+- This rule assumes that the language of each test target can be correctly determined (either programmatically or by analyzing the content), and sufficiently understood.
+
+## Accessibility Support
+
+This rule assumes that assistive technologies are exposing all links on the page in the same way no matter which [document tree](https://dom.spec.whatwg.org/#document-trees) they are in. If an assistive technology requires the user to "enter" an `iframe` or a [shadow tree][] before exposing its links, then it is possible for two links to have identical name and context but resolve to different resources without failing [Success Criterion 2.4.4 Link Purpose (In Context)][sc244] (if said links are in separate [documents][document] or [shadow trees][shadow tree])
+
+## Background
+
+- [CSS Scoping Module Level 1 (editor's draft)](https://drafts.csswg.org/css-scoping/)
+- [Understanding Success Criterion 2.4.4: Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)
+- [HTML Specification - URL parsing](https://html.spec.whatwg.org/#resolving-urls)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+A set of two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context] and link to the [same resource][].
```

Comment:
Can you make sure these example descriptions follow the format outlined here: https://github.com/act-rules/act-rules.github.io/pull/946/files#diff-9772ff397ce17764837b9ae6f1fd71acR142

Reply:
Done